packages/docs/docs/communications/async-encounters.md

```diff
@@ -0,0 +1,53 @@
+# Representing Asynchronous Encounters
+
+## Intro
+
+In healthcare, an "encounter" refers to any diagnostic or treatment interaction between a patient and provider. In traditional healthcare settings, this typically refers to an in-person visit, and is represented by the [Encounter](/docs/api/fhir/resources/encounter) resource.
+
+But in digital health, this can take on a variety of asynchronous forms, including SMS chains, in-app chat threads, or even an email exchange.
+
+In this guide, we'll show you how to represent these kinds of asynchronous encounters in FHIR.
+
+## Defining Sessions
+
+To get started, you'll first need to determine what determines a "session" in your care setting.
+
+A session could be a single SMS chain, or a single email thread. Many digital health apps ask the patient to explicitly initiate a messaging session with a provider.
+
+Alternatively, if your care setting has more of an rolling interaction model (e.g. a continuous text thread), you may choose to group all communications from the same day into a session.
+
+## Representing Sessions in FHIR
+
+Each session should be represented by an [Encounter](/docs/api/fhir/resources/encounter) resource. All of the messages that are part of this session should be represented as [Communication](#) resources and can be linked to the session using the `Communication.encounter` element.
+
+You should record the participating physicians using the `Encounter.participant` element. You can also record any family members who are part of the session here (see our [Family Relationships guide](#) ).
+
+:::tip Asynchronous Encounter Ontologies
+
+The `Encounter.class` is a required element in FHIR and should be taken from the [HL7 Act Encounter Code Valueset](https://terminology.hl7.org/3.1.0/ValueSet-v3-ActEncounterCode.html). Asynchronous care contexts will almost always use the code `VR` ("virtual").
```

Comment:
This is useful.

Reply:
One thing that I'd add here is that may want to note where CPTs should be filled in.

Reply:
May want to skim [Coded Encounters](https://app-staging.joincandidhealth.com/docs#tag/Coded-Encounters) to get a sense of which data should be included.

Reply:
Yeah, I wasn't sure how much to include in this section, as the main focus is about representing async encounters. I think. we should have a separate guide on ontologies for billing in the `Billing` section, and link from here.

Reply:
Fair enough